frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -141,7 +142,10 @@ export function DashCardVisualization({
   const dispatch = useDispatch();
 
   const editVisualization = useMemo(() => {
-    if (isVisualizerDashboardCard(dashcard)) {
+    if (
+      isVisualizerDashboardCard(dashcard) &&
+      dashboardCardSupportsVisualizer(dashcard)
+    ) {
```

Comment:
`isVisualizerDashboardCard` shouldn't be needed here since we're calling it inside `dashboardCardSupportsVisualizer` now